types/gulp-lesshint/gulp-lesshint-tests.ts

```diff
@@ -0,0 +1,18 @@
+import gulp = require('gulp');
+import lesshint = require('gulp-lesshint');
+import assert = require('assert');
+
+gulp.task('lint', () => {
+    return gulp
+        .src('./src/*.less')
+        .pipe(
+            lesshint({ configPath: './path', maxWarnings: 1 })
+                .on('data', file => {
+                    assert.strictEqual(file.lesshint.success, false);
```

Comment:
```suggestion
                    file.lesshint.success; // $ExpectType boolean;
```
(but you would have to use 'any' I think, unless you extend your interface like:

```ts
declare namespace gulpLesshint {
    interface Options {
        configPath?: string;
        maxWarnings?: number;
    }

    interface LessHintFile {
        lesshint: {
            success: boolean;
            resultCount: number;
            results: {
                column: number;
                file: string;
                fullPath: string;
                line: number;
                linter: string;
                message: string;
                severity: string;
            };
        };
        [key: string]: any;
    }

    interface Plugin {
        (options?: Options): {
            on(event: 'data', listener: (file: LessHintFile) => void): Transform;
        } & Transform;

        reporter(reporter?: string): Transform;
        failOnError(): Transform;
        failOnWarning(): Transform;
    }
}
```

Reply:
Thanks, I'll use it